job/runner.go

```diff
@@ -53,6 +53,20 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+// CleanupType for enum.
+type CleanupType int
+
+const (
+	// All is a clean up type
+	All CleanupType = iota
+	Succeeded
```

Comment:
__[Golint]__ exported const Succeeded should have comment (or a comment on this block) or be unexported ([link](https://sider.review/gh/repos/169710725/pulls/25#issue-4259318))

<sub>posted by Sider</sub>